tests/beacon/state_machines/test_proposer.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+
+from eth.constants import (
+    ZERO_HASH32,
+)
+from eth.beacon.state_machines.forks.serenity import (
+    SerenityBeaconStateMachine,
+)
+from eth.beacon.db.chain import BeaconChainDB
+
+from eth.beacon.helpers import (
+    get_block_committees_info,
+    get_new_recent_block_hashes,
+)
+
+
+@pytest.fixture
+def fixture_sm_class(config):
+    return SerenityBeaconStateMachine.configure(
```

Comment:
dropping the `Beacon` from the class name would make this a bit less of a mouthful to read: `SerenityStateMachine`

fits with the EVM naming convention since we don't name things `FrontierEVMVM` or `FrontierChainVM`